proto/spire/plugin/server/nodeattestor/v1/nodeattestor.proto

```diff
@@ -52,4 +52,9 @@ message AgentAttributes {
     // Optional. Selectors values to ascribe to the agent. The type of the
     // selectors will be inferred from the plugin name.
     repeated string selector_values = 2;
+
+    // Optional. If repeatable is true, then this attestation may happen more
+    // than one time for a given agent. This allows the server to clear out
+    // old entries automatically (since they can be easily recreated).
+    bool repeatable = 3; 
```

Comment:
I am good with any of these names.
I feel the polarity should be that if the flag is true, it should NOT be TOFU. That way if any plugins forget to fill in this field or are using old versions of this proto, then it will revert to the existing, safe, TOFU behavior.

Reply:
I agree with @dfeldman 's reasoning about the default stance being TOFU. We have to consider non-builtin NodeAttestor plugins that will not be setting this field yet. It seems preferable to me to consider them TOFU until explicitly indicated otherwise by the developer of the plugin.

Also since this field will be used to determine whether a given NodeAttestor plugin's expired data can be safely purged from the database, it is probably better to assume that a plugin is not eligible for this behavior until explicitly opted in.

@evan2645 any preference on the name? I am leaning towards `can_auto_reattest`, since it matches the desired polarity.

Reply:
Can we go ahead with can_auto_reattest?

Reply:
> I feel the polarity should be that if the flag is true, it should NOT be TOFU. That way if any plugins forget to fill in this field or are using old versions of this proto, then it will revert to the existing, safe, TOFU behavior.
> ...
>> It seems preferable to me to consider them TOFU until explicitly indicated otherwise by the developer of the plugin.
>> ...
>> Also since this field will be used to determine whether a given NodeAttestor plugin's expired data can be safely purged from the database, it is probably better to assume that a plugin is not eligible for this behavior until explicitly opted in.

All of that makes good sense!

> Can we go ahead with can_auto_reattest?

Sure. Another option could be simply `can_reattest` since evicting and agent is considered more like a fresh attestation than a re-attestation, but either is OK with me

Thank you @dfeldman!